recipes/hyper/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hyper" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "069514f54231fb7b5df2fb910a114663a83306d5296f588fffcb0a9be19407fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - h2          >=2.4,<3.0,!=2.5.0
+    - hyperframe  >=3.2,<4.0
+    - rfc3986     >=1.1.0,<2.0
+    - brotlipy    >=0.7.0,<1.0
+
+test:
+  imports:
+    - test
+
+about:
+  home: http://hyper.rtfd.org/en/latest/
+  license: MIT
```

Comment:
Also add `license_family: MIT` and more importantly `license_file: LICENSE`, please.

Reply:
(Note: done.)